### PR TITLE
Next set of status checker tool updates

### PR DESCRIPTION
**Pull request by siyer-123**
- Implemented separate installation `status` and `status-all` commands: 
  - `oc waiops status` for a more concise "standard" output
  - `oc waiops status-all` for a more detailed "verbose" output
- Reorganized installation status checker output for both commands listed above
- Added EventProcessor checks for both `status` commands and `status-upgrade`
- Updated docs and script version

Resolves https://github.ibm.com/katamari/dev-issue-tracking/issues/27751

### CODE REVIEW

**Comment by siyer-123** [comment]
Going to add the eventprocessor object in real quick before we merge this

**Comment by siyer-123** [comment]
Hi @kvijai82 @taylormgeorge91 the PR is now ready with the inclusion of the EventProcessor checks

File: kubectl-plugin/waiops-status.md [hunk 1]
```diff
@@ -259,5 +326,6 @@ chmod +x cp4waiops-samples/kubectl-plugin/kubectl-waiops
 cp cp4waiops-samples/kubectl-plugin/kubectl-waiops /usr/local/bin/kubectl-waiops
```
Comment (veronicaboychuk) [inline kubectl-plugin/waiops-status.md:326]: If the user is on IBM cloud using some different terminal, the location of `oc` might be a bit different. 
Should we suggest something like running `type oc` to determine the folder the `oc` binary is in? Or does it not matter as long as it is executable?

**Comment by taylormgeorge91** [comment]
It is a very crude "quick start" form that is not intended to cover everything. Portability is nice though. But to your other point, it's about being in the path, not necessarily where `oc` is in the path. So as long as it's executable in the path it should be found.

**Review by kvijai82 — APPROVED** [review]

**Pull request merged**